clog/builder.go

```diff
@@ -6,6 +6,9 @@ import (
 	"reflect"
 
 	"github.com/alcionai/clues"
+	"github.com/alcionai/clues/internal/stringify"
+	"go.opentelemetry.io/otel/log"
+	otellog "go.opentelemetry.io/otel/log"
```

Comment:
nit: We are importing the same package twice with different names. Can we just keep `otellog "go.opentelemetry.io/otel/log"`?

Reply:
d'oh, good catch.